src/components/scenes/TradeOptionSelectScene.tsx

```diff
@@ -0,0 +1,386 @@
+import * as React from 'react'
+import { ActivityIndicator, Image, View } from 'react-native'
+import { sprintf } from 'sprintf-js'
+
+// TradeOptionSelectScene - Updated layout for design requirements
+import paymentTypeLogoApplePay from '../../assets/images/paymentTypes/paymentTypeLogoApplePay.png'
+import { useRampQuotes } from '../../hooks/useRampQuotes'
+import { useSupportedPlugins } from '../../hooks/useSupportedPlugins'
+import { lstrings } from '../../locales/strings'
+import type {
+  RampQuoteRequest,
+  RampQuoteResult,
+  SettlementRange
+} from '../../plugins/ramps/rampPluginTypes'
+import { useSelector } from '../../types/reactRedux'
+import type { BuyTabSceneProps } from '../../types/routerTypes'
+import { getPaymentTypeIcon } from '../../util/paymentTypeIcons'
+import { getPaymentTypeDisplayName } from '../../util/paymentTypeUtils'
+import { AlertCardUi4 } from '../cards/AlertCard'
+import { PaymentOptionCard } from '../cards/PaymentOptionCard'
+import { EdgeAnim } from '../common/EdgeAnim'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { SectionHeader } from '../common/SectionHeader'
+import { styled } from '../hoc/styled'
+import { BestRateBadge } from '../icons/BestRateBadge'
+import { SceneContainer } from '../layout/SceneContainer'
+import { RadioListModal } from '../modals/RadioListModal'
+import { Shimmer } from '../progress-indicators/Shimmer'
+import { Airship } from '../services/AirshipInstance'
+import { useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+
+export interface RampSelectOptionParams {
+  rampQuoteRequest: RampQuoteRequest
+  quotes?: RampQuoteResult[]
+}
+
+interface Props extends BuyTabSceneProps<'rampSelectOption'> {}
+
+export const TradeOptionSelectScene = (props: Props): React.JSX.Element => {
+  const { route } = props
+  const { rampQuoteRequest, quotes: precomputedQuotes } = route.params
+
+  const theme = useTheme()
+  const rampPlugins = useSelector(state => state.rampPlugins.plugins)
+  const isPluginsLoading = useSelector(state => state.rampPlugins.isLoading)
+
+  // Use supported plugins hook only if no precomputed quotes
+  const { supportedPlugins } = useSupportedPlugins({
+    selectedWallet: rampQuoteRequest.wallet,
```

Comment:
Ack. just realized a short coming in our design. We never show the wallet the user is buying into or selling from. Please add a text line about Exchange Rate

"Buy into wallet: My Bitcoin"
"Sell from wallet: My Bitcoin"